apis/nucleus/src/components/listbox/ListBoxInline.jsx

```diff
@@ -301,6 +319,7 @@ function ListBoxInline({ options, layout }) {
                 onConfirm: () => {},
                 onCancel: () => {},
               }}
+              direction={isRtl}
```

Comment:
This doesn't look right? 
Could we use `direction` (rtl/ltr) for all properties passed between components and only use `isRtl` for checks witin a component? Its easier to understand if we are consistent in what properties mean.

Reply:
ok shall change